README.md

```diff
@@ -1,77 +1,225 @@
+    )
+    
+    git_repository(
+        name = "org_iota_client",
+        commit = "$CCLIENT_COMMIT_HASH",
+        remote = "https://github.com/iotaledger/iota.c.git",
+    )
 
-Second, adding it to dependencies in the **BUILD** file:  
+    # external library build rules
+    git_repository(
+        name = "rules_iota",
+        commit = "$RULES_IOTA_COMMIT_HASH",
+        remote = "https://github.com/iotaledger/rules_iota.git",
+    )
 
-```
-# The BUILD file
+    load("@rules_iota//:defs.bzl", "iota_deps")
+
+    iota_deps()
+    ```
+
+    This file tells Bazel to build the C library and its dependencies from GitHub.
+
+2. Create a `BUILD` file and add the following to it:
+
+    ```bash
+    package(default_visibility = ["//visibility:public"])
+
+    cc_binary(
+        name = "cclient_app",
+        copts = ["-DLOGGER_ENABLE"],
+        srcs = ["cclient_app.c", "cclient_app.h",],
+        deps = ["@org_iota_client//cclient:api",],
+        visibility = ["//visibility:public"],
+    )
+    ```
+
+    This file tells Bazel to build you project's source files (`cclient_app.c` and `cclient_app.h`), using the `api` package as a dependency.
+    
+Now, you're ready to start coding!
 
-package(default_visibility = ["//visibility:public"])
+## Getting Started
 
-cc_binary(
-    name = "my_app",
-    srcs = ["my_app.c", ],
-    deps = ["@org_iota_client//cclient/api",],
-    visibility = ["//visibility:public"],
-)
+After [installing the library](#installation), you can connect to an IRI node to send transactions to it and interact with the ledger.
+An extended guide can be found on our [documentation portal](https://docs.iota.org/docs/client-libraries/0.1/getting-started/c-quickstart), we strongly recommend you to go here. A quickstart tutorial is shown below.
 
+To connect to a local IRI node, you can do the following:
+
+```cpp
+#include "cclient/api/core/core_api.h"
+#include "cclient/api/extended/extended_api.h"
+
+#include <inttypes.h>
+#include "iota_client_service/config.h"
+#include "iota_client_service/client_service.h"
+
+retcode_t get_iota_node_info(iota_client_service_t *iota_client_service, get_node_info_res_t *node_response) {
+    retcode_t ret = RC_ERROR;
+    // Connect to the node
+    ret = iota_client_get_node_info(iota_client_service, node_response);
+
+    // Define variables to determine whether trit conversion succeeds
+    trit_t trytes_out[NUM_TRYTES_HASH + 1];
+    size_t trits_count = 0;
+    // If the node returned data, print it to the console
+    if (ret == RC_OK) {
+        printf("appName %s \n", node_response->app_name->data);
+        printf("appVersion %s \n", node_response->app_version->data);
+
+        // Convert the returned trits to trytes
+        // For more information about trits and trytes, see the IOTA documentation portal: https://docs.iota.org/docs/getting-started/0.1/introduction/ternary
+        trits_count = flex_trits_to_trytes(trytes_out, NUM_TRYTES_HASH,
```

Comment:
we expect user no need to take care about the conversion between flex_trits and trytes.
you can print out to the console by calling `flex_trit_print`
```
    printf("latestMilestone ");
    flex_trit_print(node_response->latest_milestone, NUM_TRITS_HASH);
    printf("\n");
```